File: load_redis/gao.py
#!/usr/bin/env python

import json
import re

# objective
#   created_at: timestamp
#   tweet id:
#   text:


def chaogao(tweet_json):
    #print tweet_json
    tweet = json.loads(tweet_json)
    #text  = (repr(tweet['text']))
    #if(text[0] == 'u'):
        #text = text[2:-1]
    #text = text.replace('/','\\/')
    i = tweet_json.find("text\":\"") + 7
    pattern = re.compile(r'[^\\]"')
    m = pattern.search(tweet_json, i)
    j = m.end()-1
    text = tweet_json[i:j]
    print "{}:{}".format(tweet['id'], text)
    #print ""

def gao(f):
    for line in f:
        if line.strip() != "":
            chaogao(line.rstrip())

gao(open('small.json', "r"))

File: load_redis/gao.js
var fs = require('fs');

function gao(filename) {
  var data = fs.readFileSync(filename, 'utf8');
  var content = data.split('\n').filter(function (line) {
    return line;
  });
  for (var i=0; i < content.length; ++i) {
    var tweet = JSON.parse(content[i]);
    var ts = Date.parse(tweet['created_at']) / 1000;
    console.log(ts);
  }
}

gao('small.json');
